state_machine/applications/system-check/test_utils.py

```diff
@@ -0,0 +1,54 @@
+try:
+    from ulab import numpy
+except ImportError:
+    import numpy
+
+norm = numpy.linalg.norm
+
+less_in_magnitude = (
```

Comment:
took me a while to figure out what these are (and you've described them to me a bit). At a minimum could you end the name of all of the things that go in the "condition" field of expect() with `_condition`. 

For example:
`less_in_magnitude_condition`